tests/MockAdapter.ts

```diff
@@ -0,0 +1,10 @@
+import MockAdapter from "axios-mock-adapter";
+import { AxiosAdapter } from "../src/Adapters";
+
+export class AxiosMockAdapter extends AxiosAdapter {
+    public mockAdapter: MockAdapter
+    constructor(baseURL: string) {
+        super(baseURL);
+        this.mockAdapter = new MockAdapter(this.client, { onNoMatch: 'throwException' });
+    }
+}
```

Comment:
Install `EditorConfig.EditorConfig` and format again, should fix the missing ending newline.

Reply:
added rule to eslint config instead